test/docs/nested_test.rb

```diff
@@ -131,6 +131,36 @@ class JsonValidate < Validate
     C::Create.(seq: [], params: nil).inspect(:seq).must_equal %{<Result:true [[:create, :validate, :json, :save]] >}
   end
 
+  it "Nested(:method), with pass_fast returned from nested" do
+    class JustPassFast < Trailblazer::Operation
+      step :just_pass_fast, pass_fast: true
+      include T.def_steps(:just_pass_fast)
+    end
+
+    module D
+
+      create =
+      #:nested-with-pass-fast
+      class Create < Trailblazer::Operation
+
+        def compute_nested(ctx, **)
+          JustPassFast
+        end
+
+        step :create
+        step Nested(:compute_nested)
+        step :save
+        #~meths
+        include T.def_steps(:create, :save)
+        #~meths end
+      end
+      #:nested-with-pass-fast end
+
+      # pass fast
+      create.(seq: []).inspect(:seq).must_equal %{<Result:true [[:create, :just_pass_fast, :save]] >}
```

Comment:
why you assign it to a variable instead of using the Class directly ?

Reply:
got "inspired" by other tests above, I can use operation class directly here I guess

Reply:
It fine. ship it

Reply:
nice :+1: